README.md

```diff
@@ -22,7 +22,7 @@ Please note that the example app only demonstrates a subset of the SDK features.
 ---
 **NOTE**
 
-QR code reading is not available at the moment.
+QR code scanning in the Example Application is only available for the Old Architecture at the moment as we have not found any open source library that supports both. If you want to support this feature for the New Architecture, you can find alternatives that support it.
 
 ---
```

Comment:
Please reformat this to use the conventional NOTE format for a markdown file:

```
> **Note**
> QR code scanning in the Example Application is only available for the Old Architecture at the moment as we have not found any open source library that supports both. If you want to support this feature for the New Architecture, you can find alternatives that support it.
```

You can see example for this below, and [here](https://github.com/orgs/community/discussions/16925) is the corresponding "official" discussion.

Reply:
Unfortunately this is not part of Markdown itself, but GitHub specific and it already changed since the docs you link was written. Anyway, I updated it to the [current method](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).